src/Libraries/CoreNodesUI/CoreNodesUI.csproj

```diff
@@ -81,7 +81,6 @@
     <Compile Include="UI\ElementSelectionControl.xaml.cs">
       <DependentUpon>ElementSelectionControl.xaml</DependentUpon>
     </Compile>
-    <Compile Include="UI\DropDownNodeViewCustomization.cs" />
```

Comment:
I think we don't want to remove this file (even if it is redundant) because it's not related to re-execution. Please bring it back.